src/reaper_osara.cpp

```diff
@@ -3782,6 +3782,8 @@ Command COMMANDS[] = {
 	{MIDI_EVENT_LIST_SECTION, {{0, 0, 40835}, NULL}, NULL, cmdMidiMoveToTrack}, // Activate next MIDI track
 	{MIDI_EDITOR_SECTION, {{0, 0, 40836}, NULL}, NULL, cmdMidiMoveToTrack}, // Activate previous MIDI track
 	{MIDI_EVENT_LIST_SECTION, {{0, 0, 40836}, NULL}, NULL, cmdMidiMoveToTrack}, // Activate previous MIDI track
+	{MIDI_EVENT_LIST_SECTION, {{0, 0, 40836}, NULL}, NULL, cmdMidiMoveToTrack}, // Activate previous MIDI track
```

Comment:
It looks like we already had a duplicate of 40836 and this adds another duplicate. :) Can you remove both please?

Reply:
duplicate removed.  The existing duplicate isn't really a duplicate, it is necessary for the command to work in the event list,  However it is not currently assigned in the keymap for the event list.